packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -250,7 +250,6 @@ registry.registerOp(CalibrateEnsembleCiemssOp);
 registry.registerOp(ModelConfigOp);
 registry.registerOp(CalibrateCiemssOp);
 registry.registerOp(DatasetTransformerOp);
-registry.registerOp(CodeAssetOp);
```

Comment:
If we are going to remove code asset in the workflow do we have a plan for removing `packages/client/hmi-client/src/components/workflow/ops/code-asset`?